### PR TITLE
[CARMAN-188] Grammar Improvements

### PR DESCRIPTION
**Pull request by daarbelaez**
## Jira ticket
[CARMAN-188](https://danchez.atlassian.net/browse/CARMAN-188)

### Description
Grammar and clarity improvements for the Privacy Policy section on the website.

### Evidence
Screenshots or videos of the app with the new features.

### Depends on
Link to pull requests that are needed for this PR. If this PR depends on other PR, please add Don’t merge label.

### CODE REVIEW

File: pages/privacypolicy.en.html [hunk 1]
```diff
@@ -1,217 +1,317 @@
...
+                The use of cookies - or other tracking tools - by this application or by the owners of third-party
+                services used by this application is for the purpose of providing the service requested by the user, as
+                well as any other purposes described in this document and in the cookie policy.<br><br>
+                The user assumes responsibility for the personal data of third parties obtained, published, or shared
+                through this application.
+            </p>
+
+            <h3>Mode and location of data processing for the collected data</h3>
+
+            <h4>Processing methods</h4>
```
Comment (dan-chez) [inline pages/privacypolicy.en.html:72]: Maybe add a - at the beginning(?)

File: pages/privacypolicy.en.html [hunk 2]
```diff
@@ -1,217 +1,317 @@
...
+                The data processing will be carried out using computers and/or IT tools, following procedures and
+                organizational methods strictly related to the purposes indicated. In addition to the data controller,
+                in some cases, certain categories of authorized persons related to the operation of this application
+                (administration, sales, marketing, legal, and system administration departments) or external contractors
+                providing services to the data controller (such as external technical service providers, courier
+                companies, hosting companies, IT companies, communication agencies) may have access to the data, as
+                appointed by the data controller as Data Processors, if necessary. An updated list of such individuals
+                can be requested from the data controller at any time.</p>
+
+            <h4>Location</h4>
```
Comment (dan-chez) [inline pages/privacypolicy.en.html:85]: Maybe add a - at the beginning(?)

File: pages/privacypolicy.en.html (hunk 1 above)
Comment (dan-chez) [reply pages/privacypolicy.en.html:72]: If you are agree, it should be add it to spanish version.

Comment (daarbelaez) [reply pages/privacypolicy.en.html:72]: We would have to add them to all subtitles and it seems to me that without the - is fine.

**Pull request merged**